src/main/java/com/emc/object/s3/jersey/S3JerseyClient.java

```diff
@@ -605,6 +605,11 @@ public void deleteObject(String bucketName, final String key) {
         executeAndClose(client, new S3ObjectRequest(Method.DELETE, bucketName, key, null));
```

Comment:
change the old `delete()` object method to wrap the new `deleteObject()` method, similar to other wrapper methods here, like https://github.com/EMCECS/ecs-object-client-java/blob/master/src/main/java/com/emc/object/s3/jersey/S3JerseyClient.java#L541 and https://github.com/EMCECS/ecs-object-client-java/blob/master/src/main/java/com/emc/object/s3/jersey/S3JerseyClient.java#L569

Reply:
Good point, fixed.